### ISSUE TITLE
Math support 

### ISSUE THREAD

**Issue by Opussci** [inokawa/remark-docx#4]
Would be nice if LaTeX math could be exported to docx. Are there any plans to go in that direction?

**Comment by inokawa** [comment]
Hi, 
remark has [remark-math](https://github.com/remarkjs/remark-math) plugin to keep math syntax and openxml actually has OfficeMath element.
It would be possible to convert some latex to docx so I'll try to make minimum feature at first.

**Comment by Opussci** [comment]
Yes, that would work. If there is online presentation than docx conversion would be useful also.
Thank you.

**Comment by inokawa** [comment]
@Opussci 
I added basic LaTeX support in `0.1.0`.
It might have some bugs and have missing features.

Any feedbacks are welcomed.
Thanks!

**Comment by Opussci** [comment]
Thank you! I will test it and get back to you.

**Comment by Opussci** [comment]
@inokawa
First impression are:

Inline and simple block display is working, labels are not active nor rendered. 

Equation and align environments are not rendered properly.

Still, amazing speed with which you implemented this.

Do you can suppose how much time will be needed to implement completely math rendering?

Best!

**Comment by inokawa** [comment]
I think perfect implementation of LaTeX is impossible because there are features that doesn't exist in Office Open XML, many of them could be imitated though.

The needed time is case by case. Some features are easy to implement and the others might be difficult.

Thank you for your feedback.

**Comment by Opussci** [comment]
Yes of course, I was thinking about major LaTeX features, like equations with numbering and cross-referencing. I think that would be more than enough for most of the cases.

Thank you for your work and short time response.

**Comment by Strengthless** [comment]
@inokawa Is there any way I can export numbered lists in alphabetical style?

What I have currently achieved so far:
![image](https://user-images.githubusercontent.com/40143583/208977381-db86a03e-7acb-4244-8df5-14050c3094f3.png)
Expected behaviour:
![image](https://user-images.githubusercontent.com/40143583/208977453-76273c6e-dd69-4efd-acda-9d093af46665.png)

**Comment by inokawa** [comment]
Completely rewritten in `0.3.12` for much more reliable output.
